pmix.sty

```diff
@@ -853,6 +853,12 @@
 % convenience macro for formatting the word "Note:" at the beginning of note blocks:
 \newcommand{\noteheader}{{\textrm{\textsf{\textbf\textup\normalsize{{{{Note: }}}}}}}}
 
+% Return values
+\newcommand{\returnstart}{Returns PMIX_SUCCESS or one of the following error codes when the condition described occurs:}
+\newcommand{\returnend}{If none of the above return codes are appropriate, then an implementation must return either a general \ac{PMIx} error code or an implementation defined error code as described in Section \ref{api:struct:errors}.}
+\newcommand{\returnsimple}{Returns \refconst{PMIX_SUCCESS} or a negative value indicating the error.}
+\newcommand{\returnsimplenb}{A successful return indicates that the request is being processed by the host environment and the result will be returned in the provided \refarg{cbfunc}. Note that the library must not invoke the callback function prior to returning from the \ac{API}.  In all other cases, the \refarg{cbfunc} will \textit{not} be called.}
```

Comment:
Revision Exception:

```suggestion
\newcommand{\returnsimplenb}{A successful return indicates that the request is being processed and the result will be returned in the provided \refarg{cbfunc}. Note that the library must not invoke the callback function prior to returning from the \ac{API}.  In all other cases, the \refarg{cbfunc} will \textit{not} be called.}
```

Rationale since this makes it more implementation agnostic on what PMIx entity is doing the processing of the request.